.github/workflows/R-CMD-check.yaml

```diff
@@ -47,6 +47,10 @@ on:
         type: boolean
         default: true
         required: false
+      upload-snapshots: 
+        type: boolean
+        default: false
```

Comment:
Should we make this `true` by default?

We do not pay for space as our repos are public

Reply:
```suggestion
        default: true
```